qubes_config/global_config.glade

```diff
@@ -6369,7 +6274,7 @@ Documentation will open in a disposable qube.</property>
                               <object class="GtkLabel" id="url_info">
                                 <property name="visible">True</property>
                                 <property name="can-focus">False</property>
-                                <property name="label" translatable="yes">To learn more about Open URL policy and its uses, you can check the &lt;a href="https://github.com/Qubes-Community/Contents/blob/master/docs/common-tasks/opening-urls-in-vms.md"&gt;online documentation&lt;/a&gt;. 
+                                <property name="label" translatable="yes">To learn more about Open URL policy and its uses, you can check the &lt;a href="https://github.com/Qubes-Community/Contents/blob/master/docs/common-tasks/opening-urls-in-vms.md"&gt;online documentation&lt;/a&gt;.
```

Comment:
Will change depending on the answer to https://github.com/QubesOS/qubes-desktop-linux-manager/pull/150#issuecomment-1529486150.